core/src/main/resources/reference.conf

```diff
@@ -374,15 +371,15 @@ docker {
     max-time-between-retries = 1 minute
     max-retries = 3
 
-    // Supported registries (dockerhub, gcr, gar, quay) can have additional configuration set separately
-    gcr {
+    // Supported registries (Docker Hub, Google, Quay) can have additional configuration set separately
+    google {
```

Comment:
Is there Terra config that will need to be updated?

Reply:
Nope, Terra did not configure any of this and is falling back on the bundled defaults.